src/js/components/pages.js

```diff
@@ -348,3 +352,53 @@ export class Service {
 }
 
 export class ServicePointPage extends Service {}
+
+export class NoticeIndex {
+  constructor(content) {
+    this.breadcrumbItems = getBreadcrumbsWithLabel(content.ancestor_pages);
+    this.childPages = content.child_pages;
+  }
+
+  renderChildPageLinks() {
+    const childPageLinks = this.childPages.map((page) => {
+      const props = {
+        title: page.title,
+        url: page.url,
+        subtitle: "",
+      };
+
+      return new LinkBlock(props);
+    });
+
+    if (childPageLinks.length) {
+      return [new FullWidthGrid(childPageLinks).render()];
+    } else {
+      return [];
+    }
+  }
+
+  render() {
+    return [
+      new Breadcrumbs(this.breadcrumbItems).render(),
+      ...this.renderChildPageLinks(),
+    ];
+  }
+}
+
+export class NoticePage {
+  constructor(content) {
+    this.breadcrumbItems = getBreadcrumbsWithLabel(content.ancestor_pages);
+    this.noticeMainContent = content.body_html;
+    this.publicationDate = content.publication_date;
+  }
+
+  render() {
+    return [
+      new Breadcrumbs(this.breadcrumbItems).render(),
+      new SectionHeading("Overview").render(),
+      new ExpandableRichText(this.noticeMainContent).render(),
```

Comment:
Let's use the non-expanding version of this - it should be in the component library - I don't think collapsing makes sense for notices that are like microblogs